server/views/pages/alpha/activityList/selectActivityLocation.njk

```diff
@@ -0,0 +1,89 @@
+{% extends "partials/layout.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "components/datePicker/datePicker.njk" import datePicker %}
+
+{% set title = "View by activity or appointment location" %}
+
+{% block content %}
+    <h1 class="govuk-heading-l">{{ title }}</h1>
+
+    {% if errors.length > 0 %}
+        {{ govukErrorSummary({
+            titleText: "There is a problem",
+            errorList: errors,
+            attributes: { 'data-qa-errors': true }
+        }) }}
+    {% endif %}
+
+    <form action="/manage-prisoner-whereabouts/select-location" method="POST">
```

Comment:
If the URL is the same as the `GET`, which it should, then you dont need to provide `action` attribute again
```suggestion
    <form method="POST">
```

Reply:
Ah, this was the wrong url anyway 😉

Reply:
Will bear this in mind - havent started on this bit yet